### DISCUSSION TITLE
Welcome to luaunit Discussions!

### DISCUSSION THREAD

**Discussion by bluebird75** [bluebird75/luaunit#136] [General]
## 👋 Welcome!

Discussions is the perfect place to talk about LuaUnit, without necessarily open an issue.

Please share your LuaUnit problems or success stories, I'll be more than happy to know about it.

**Comment by bluebird75** [comment]
I recently published an article in french on [linuxfr](https://linuxfr.org/users/bluebird/journaux/combien-d-utilisateurs-pour-luaunit) to talk about LuaUnit success.

The most interesting part of it is that there are about 1100 projects on GitHub which take advantage of LuaUnit !

Thanks to all contributors !

**Comment by mikerieser** [comment]
Hello Philippe,

I recently discovered and ported luaunit.lua to work in the sandboxed environment of Tabletop Simulator (TTS). By the way, TTS uses Moonsharp 2.0.0.0 as its Lua engine (v 5.2 Lua compatible). I was wondering if you want any of what I've done?
I might also have a feature request. I need support for another type of Output. TextOutput works, but the behavior of `print()` in TTS is to append with a newline. File output is impossible in the sandboxed environment. I'm working toward an Output that will update a graphical display, but I've found it difficult to override the current behavior cleanly. Let me know your level of interest in this, and I'd be happy to show you more.
